app/src/main/java/com/example/fivenightsescape/MapsActivity.kt

```diff
@@ -128,8 +133,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
         }
     }
 
-    private fun gameInit(currentLatLng: LatLng)
-    {
+    private fun gameInit(currentLatLng: LatLng) {
```

Comment:
instead style it how it was before for functions:

private fun functionName()
{
    // code
}

instead of 

private fun functionName() {
    // code
}